standard/ERCs/erc-6900.md

```diff
@@ -171,35 +209,13 @@ interface IStandardExecutor {
     /// @param calls The array of calls.
     /// @return An array containing the return data from the calls.
     function executeBatch(Call[] calldata calls) external payable returns (bytes[] memory);
-}
-```
-
-#### `IModuleExecutor.sol`
-
-Execution interface for calls made from modules. Modular Smart Contract Accounts **MUST** implement this interface to support execution from modules.
-
-The `executeFromModuleExternal` function SHOULD check whether the call's target implements the `IModule` interface via ERC-165.
 
-**If the target of `executeFromModuleExternal` function is a module, the call SHOULD revert.**
-
-This prevents accidental misconfiguration or misuse of modules (both installed and uninstalled). Installed modules MAY interact with other installed modules via the `executeFromModule` function.
-
-```solidity
-interface IModuleExecutor {
-    /// @notice Execute a call from a module through the account.
-    /// @dev Permissions must be granted to the calling module for the call to go through.
+    /// @notice Execute a call using a specified runtime validation, as given in the first 21 bytes of
```

Comment:
This should be a `bytes24` now, since we changed `uint8 functionId` to `uint32 entityId`.